doc/en/how-to/usage.rst

```diff
@@ -76,11 +76,19 @@ Specifying a specific parametrization of a test:
 
 **Run tests by marker expressions**
 
+To run all tests which are decorated with the ``@pytest.mark.slow`` decorator:
```

Comment:
```suggestion
To run all tests which are decorated with the ``@pytest.mark.slow`` decorator, execute the following command:
```

Reply:
The suggestion is incoherent with previous examples, leaving as is.